session_3/blogspot/README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
Please describe the things you have implemented and add screenshots. For example: https://github.com/VikramSGIT/IRIS-RoR-Bootcamp-2020/tree/session_4_vikram/session_4/blogspot

Reply:
Does it have to be as well designed as this repo? OP has added photo support, and styles on the project, which I haven't done.

Reply:
The only changes I have made as of now are regarding redirects XD. After logging in and logging out, I have made it so that they are redirected to the main site

Reply:
As you add new features, document them. Even if they are not "bonus" tasks as at least I can understand what to expect before reading code.